devel/example_workflow.R

```diff
@@ -47,11 +53,8 @@ setup(git_stats,
 get_repos(git_stats)
 
 # you can plot repos sorted by last activity
-
 plot_repos(git_stats)
```

Comment:
There's something wrong with this function as the plot I get looks like that: 
![image](https://github.com/r-world-devs/GitStats/assets/32685910/98bdfd42-0897-42a2-ba50-3515bb33db40)
 but when you compare it with the code that was on this #212 PR you may see the difference in the code. Like all the changes made for this function didn't get into devel branch at all.

Reply:
I handled it - the root cause was not in `plot_repos` but in pulling repos by team which was recently refactored with pulling repos by user directly via GraphQL in #214 - in some cases the repos were pulled multiple times (for many user same repos) - and that distorted the plot. The solution was simply applying distinct rows to output repos table.

Reply:
Oh, that would explain double rows, indeed. Looks good now! :)